Userland/Makefile

```diff
@@ -4,7 +4,7 @@ APPS = ${SRCS:.cpp=}
 
 EXTRA_CLEAN = $(APPS)
 
-LIB_DEPS = Web GUI Gfx Audio Protocol IPC Thread Pthread PCIDB Markdown JS Core Line X86 Debug
+LIB_DEPS = Crypto TLS Web GUI Gfx Audio Protocol IPC Thread Pthread PCIDB Markdown JS Core Line X86 Debug
```

Comment:
Possibility : Should we add HTTP as a precaution, or just wait until it's required ?

Reply:
There's no particular need for HTTP, at least, not until someone (re)develops `curl` or whatever